edx_django_utils/monitoring/internal/utils.py

```diff
@@ -112,6 +112,16 @@ def function_trace(function_name):
         yield
 
 
+def set_monitoring_transaction_name(name, group=None, priority=None):
+    """
+    Sets the transaction name for monitoring.
+
+    This is not cached.
```

Comment:
```suggestion
    Sets the name, group, and priority for the current root span.

    Current root span refers to the most ancestral span in the current process, rather than
    to the trace root, which may be outside of the process.

    Group and priority may not be supported by all backends.
```
(I suspect the "not cached" note is an artifact that is no longer meaningful. At least, I have no idea what it means here...)